app/src/main/java/org/simple/clinic/editpatient/EditPatientEffectHandler.kt

```diff
@@ -131,22 +131,18 @@ object EditPatientEffectHandler {
       dateOfBirthFormatter: DateTimeFormatter,
       utcClock: UtcClock
   ): Patient {
-    val patientWithoutAgeOrDateOfBirth = patient.copy(
-        fullName = ongoingEntry.name,
-        gender = ongoingEntry.gender,
-        dateOfBirth = null,
-        age = null
-    )
+    val patientWithoutAgeOrDateOfBirth = patient
+        .withoutAgeOrDateOfBirth(ongoingEntry.name, ongoingEntry.gender)
```

Comment:
I think this naming is a little tricky because the name of the function and the parameters we are passing do not make sense with each other. What do you think of adding two more functions to transform the `Patient` so that this call would look more like:

```kotlin
val patientWithoutAgeOrDateOfBirth = patient
  .withNameAndGender(ongoingEntry.name, ongoingEntry.gender)
  .withoutAgeAndDateOfBirth()
```